### PR TITLE
functionality updates

### PR DESCRIPTION
**Pull request by AsteriHoku**
main readme updates
adding non-euler contributions section to main readme
adding rainbowstring and binarystring functions in single file as module exports
adding colors package for fun printing for language challenges
turned unused_langs into a script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,18 +107,21 @@ npm run euler
...
-or `node index.js`. This script will create all of the correct files and 
-folders, and will create a README file in your solution directory.
+### 5. Code Your Solution (for each problem)
+Start a new Project Euler solution by 1 of 2 options:
+
+   * run the command `npm run euler` 
+or `node index.js`
+
+   * provide command-line arguments to the script to skip 
+interactive mode<br>Examples:
```
Comment (blister) [inline README.md:117]: Why did you add an HTML line break here?

**Review by blister — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AsteriHoku) [reply README.md:117]: To put the Examples on a new line. I can send you a screenshot.
